ep_gui/resources/lang/ep_gui.en-US.rc

```diff
@@ -341,10 +341,10 @@ BEGIN
 
     IDS_OTHER                   "Other"
 
-    IDS_OTHER_REMEMBERLAST      "Remember last used section in this window"
+    IDS_OTHER_REMEMBERLAST      "Open ExplorerPatcher on the tab of the last used section"
     IDS_OTHER_CLOCKFLYOUT       "Open clock flyout when pressing Win+C instead of Microsoft Teams"
     IDS_OTHER_TOOLBARSEPARATORS "Show separators between taskbar toolbars"
-    IDS_OTHER_WINXPROPERTIES    "Add shortcut to program settings in Win+X menu"
+    IDS_OTHER_WINXPROPERTIES    "Add shortcut to ExplorerPatcher settings in Win+X menu"

```

Comment:
"Add ExplorerPatcher shortcut to settings in Win+X menu" is correct.

Reply:
OK